waku/v2/node/waku_node.nim

```diff
@@ -47,8 +47,10 @@ when defined(rln):
   import
     ../waku_rln_relay
 
-declarePublicGauge waku_version, "Waku version info (in git describe format)", ["version"]
 declarePublicCounter waku_node_messages, "number of messages received", ["type"]
+declarePublicHistogram waku_histogram_message_size, "message size histogram in kB", buckets = [0.0, 5.0, 15.0, 50.0, 100.0, 300.0, 700.0, 1000.0]
```

Comment:
Perhaps I'm missing something about how `nim-metrics` work, but should the `Inf` bucket be declared?

Reply:
Indeed, added in e27702c89b8ed7cc06d67bd2a6f029915e029195 to be explicit.

But looks like its automatically added. Have seen examples that both provide `Inf` and not, but seems that even without it its automatically added. Example without having `Inf` in bucket.

```console 
$ curl localhost:8004/metrics | grep histogram
...
waku_histogram_message_size_sum 0.0
waku_histogram_message_size_count 0.0
waku_histogram_message_size_created 1681987077.0
waku_histogram_message_size_bucket{le="0.0"} 0.0
waku_histogram_message_size_bucket{le="5.0"} 0.0
waku_histogram_message_size_bucket{le="15.0"} 0.0
waku_histogram_message_size_bucket{le="50.0"} 0.0
waku_histogram_message_size_bucket{le="100.0"} 0.0
waku_histogram_message_size_bucket{le="300.0"} 0.0
waku_histogram_message_size_bucket{le="700.0"} 0.0
waku_histogram_message_size_bucket{le="1000.0"} 0.0
waku_histogram_message_size_bucket{le="+Inf"} 0.0   <---
```

Reply:
Thanks! I thought it may be added automatically, but wasn't sure.